tests/integration/test_client_properties.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from bumble.device import Device
+
+from bleak import BleakClient
+from bleak.backends import BleakBackend, get_default_backend
+from tests.integration.conftest import (
+    configure_and_power_on_bumble_peripheral,
+    find_ble_device,
+)
+
+
+@pytest.mark.skipif(
+    get_default_backend() != BleakBackend.CORE_BLUETOOTH,
+    reason="'get_rssi()' is only available on CoreBluetooth",
+)
+async def test_get_rssi(bumble_peripheral: Device):
+    """Getting RSSI from client is possible."""
+    await configure_and_power_on_bumble_peripheral(bumble_peripheral)
+
+    device = await find_ble_device(bumble_peripheral)
+
+    async with BleakClient(device) as client:
+        rssi = await client._backend.get_rssi()  # type: ignore
```

Comment:
```suggestion
        rssi = await client._backend.get_rssi()  # pyright: ignore[reportPrivateUsage]
```

Makes it more obvious what exactly we are ignoring.